sync/librariesio.js

```diff
@@ -143,13 +240,16 @@ for (let page of pages) {
     }
 
     if (!debugMode) {
-      json_output({
-        id_prefix: libinfo[lang].prefix,
+      name = name.replace(/.+[/]/, ''); // strip path from name
+      repo = repo.replace(/^.*github\.com\//, ''); // stip github.com from repo
```

Comment:
It's nitpicking, but our standard has always been complete sentences for comments. Including periods in the end. Here and elsewhere.
`// Strip github.com from repo.`